datafusion/sqllogictest/test_files/join.slt.part

```diff
@@ -1504,7 +1504,7 @@ drop table t0;
 statement ok
 create table t1(v1 int, v2 int);
 
-query error DataFusion error: Schema error: No field named tt1.v2. Valid fields are tt1.v1.
+query error DataFusion error: Error during planning: Column in ORDER BY must be in GROUP BY or an aggregate function
```

Comment:
This is changed now because of `check_columns_satisfy_exprs` -- I think this is a better error IMO vs what was there.